data.sql

```diff
@@ -5,3 +5,12 @@ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_
   (2, 'Gabumon', '2018-11-15', 2, true, 8),
   (3, 'Pikachu', '2021-01-07', 1, false, 15.04),
   (4, 'Devimon', '2017-05-12', 5, true, 11);
+
+  INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES 
+  (5, 'Charmander', '2020-02-08', 0, false, -11),
+	(6, 'Plantmon', '2022-11-15', 2, true, -5.7),
+	(7, 'Squirtle', '1993-04-02', 3, false, -12.13),
+  (8, 'Angemon', '2005-06-12', 1, true, -45),
+  (9, 'Boarmon', '2005-06-07', 7, true, 20.4),
+  (10, 'Blossom', '1998-08-13', 3, true, 17)
```

Comment:
- [ ] Please add Ditto and its details to the animals table. No Digimon left behind. You also did not close this command with a semi-colon.
![image](https://github.com/Khaled-Alaa-1/Vet-clinic/assets/45460192/521730bf-16b9-4172-b3b6-911f9351e436)